addons/europeans_tribe.wad/ship/init.lua

```diff
@@ -106,6 +106,32 @@ descriptions:new_ship_type {
       -- TRANSLATORS: This europeans ship is named after a continent
       pgettext("shipname", "Europa"),
       pgettext("shipname", "Europe"),
+      pgettext("shipname", "Island"),
```

Comment:
I guess you mean the country with the German name Island and the English name:
```suggestion
      pgettext("shipname", "Iceland"),
```

Reply:
Most names are fictious derivation of real names. -> Balkania or Polandia. I like to have names which ends with a "a".